### PR TITLE
feat: add ministral 3b and 8b and some small enhancements

### PR DESCRIPTION
**Pull request by Wendong-Fan**
## Description

- use anthropic types for anthropic model
- add dependencey requried decorator for models part
- add mistral 3b and 8b
- flexible pytorch version for poetry install
- rename output_schema to response_format

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/camel-ai/camel/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [ ] Example (update in the folder of example)

## Implemented Tasks

- [ ] Subtask 1
- [ ] Subtask 2
- [ ] Subtask 3

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [CONTRIBUTION](https://github.com/camel-ai/camel/blob/master/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: camel/types/enums.py [hunk 1]
```diff
@@ -265,6 +269,8 @@ def token_limit(self) -> int:
             ModelType.MISTRAL_LARGE,
             ModelType.MISTRAL_NEMO,
             ModelType.MISTRAL_PIXTRAL_12B,
+            ModelType.MISTRAL_8B,
+            ModelType.MISTRAL_3B,
```
Comment (lightaime) [inline camel/types/enums.py:273]:
```suggestion
            ModelType.MISTRAL_MINISTRAL_3B,
            ModelType.MISTRAL_MINISTRAL_8B,
```

Comment (lightaime) [reply camel/types/enums.py:273]: Sorry it seems to fine to ccall `ModelType.MISTRAL_3B`. Can we move it up and sort by the names?

Comment (Wendong-Fan) [reply camel/types/enums.py]: updated

**Comment by lightaime** [comment]
Sorry it seems to fine to call ModelType.MISTRAL_3B. They call it this way in their apis. Please ignore the previous review. Can we move it up and sort by the names?

**Review by lightaime — APPROVED** [review]
Can we followup a PR with an example for using `MISTRAL_3B` and `MISTRAL_8B` for function call and make a post about it?

**Pull request merged**